src/cutscenes/intro.inc.h

```diff
@@ -29,7 +29,7 @@
 		T_REIMU("Normally it *would* be comforting, but…"),
 		{ 0 },
 	}},
-	{ "cutscenes/reimu_bad/01", {
+	{ "cutscenes/opening/01", {
```

Comment:
Why was this changed? I actually intended to reuse a shot from the bad ending here, with Reimu and a crowd in front of the shrine. Does it not make sense?